.github/workflows/python-publish.yml

```diff
@@ -21,11 +21,11 @@ jobs:
       - name: Install dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install setuptools twine
+          python -m pip install build twine
       - name: Build and publish
         env:
           TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
           TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
         run: |
-          python setup.py sdist
```

Comment:
Moved away from `setup.py` invocation. `python -m build .` will build both the sdist and wheel automatically under `dist`.

https://blog.ganssle.io/articles/2021/10/setup-py-deprecated.html

Reply:
Moved back to setup.py